bitnami/common/templates/_affinities.tpl

```diff
@@ -47,17 +47,21 @@ Return a nodeAffinity definition
 
 {{/*
 Return a soft podAffinity/podAntiAffinity definition
-{{ include "common.affinities.pods.soft" (dict "component" "FOO" "context" $) -}}
+{{ include "common.affinities.pods.soft" (dict "component" "FOO" "extraMatchLabels" .Values.extraMatchLabels "context" $) -}}
```

Comment:
Hi @kuznero ,
Thank you very much for all the changes!

Did you check that this will continue working with the rest of charts that are not passing the `extraMatchLabels`?

Reply:
Yes, I set it to empty `dict` by default. Checked it with `redis` chart.